lru_cache/lru_cache.py

```diff
@@ -29,5 +35,9 @@ def get(self, key):
     want to overwrite the old value associated with the key with
     the newly-specified value.
     """
+
     def set(self, key, value):
-        pass
+        # add key, value pair to cache
```

Comment:
Planning, nice your usage of commits showcases how you plan around in building your solution.